includes/derivatives.inc

```diff
@@ -34,6 +34,7 @@ function islandora_handle_append_handle_from_configuration(AbstractObject $objec
       $handle_exists = TRUE;
     }
     else {
+      $error = $response->error;
```

Comment:
Stupidly minor, but it's not really necessary for us to store this variable in a second location since it's already a property of `$response`; may as well use `$response->error` as the substitution for `@error` down in `message_sub` below